usr/sbin/p11sak/p11sak.c

```diff
@@ -2663,6 +2914,13 @@ int main(int argc, char *argv[])
         goto done;
     }
 
+    /* Parse p11sak_defined_attrs.conf */
+    rc = parse_file();
+    if (rc != CKR_OK) {
+        printf("Failed to open and/or parse p11sak_defined_attrs.conf (error code 0x%lX: %s)\n", 
```

Comment:
maybe omit the file name (p11sak_defined_attrs.conf) in this error message, as `parse_file()` already prints a message with the real file name. The file name may be overridden by user by environment variable, so this hard coded file name may be wrong/missleading. Maybe you can even omit the whole error message here, if it is guaranteed that `parse_file()` already prints one in all error cases.

Reply:
Done